src/legacy/core_plugins/kbn_vislib_vis_types/public/pie.js

```diff
@@ -80,7 +80,7 @@ export default function HistogramVisType(Private) {
         {
           group: 'buckets',
           name: 'segment',
-          icon: 'fa fa-scissors',
+          icon: 'cut',
           title: i18n.translate('kbnVislibVisTypes.pie.segmentTitle', { defaultMessage: 'Split Slices' }),
```

Comment:
Where you see the titles of these options, can you make sure they're sentence case? IE: `Split Slices` -> `Split slices`

Reply:
What about dashed ones: e.x. `X-Axis` ?

Reply:
Should be lowercase as well: `X-axis`